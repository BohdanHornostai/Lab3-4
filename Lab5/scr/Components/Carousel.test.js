import React from 'react'
import {render, cleanup} from '@testing-library/react'
import { Carousel } from 'react-bootstrap'

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<Carousel />)

    expect(asFragment(<Carousel />)).toMatchSnapshot()
   })