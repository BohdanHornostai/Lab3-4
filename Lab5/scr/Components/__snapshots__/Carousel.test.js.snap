// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="carousel slide"
  >
    <ol
      class="carousel-indicators"
    />
    <div
      class="carousel-inner"
    />
    <a
      class="carousel-control-prev"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-prev-icon"
      />
      <span
        class="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      class="carousel-control-next"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-next-icon"
      />
      <span
        class="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</DocumentFragment>
`;
